archive/opentelemetry-browser-extension-autoinjection/package.json

```diff
@@ -57,7 +57,7 @@
     "sinon-chrome": "3.0.1",
     "ts-loader": "9.2.5",
     "ts-mocha": "10.0.0",
-    "typescript": "4.3.5",
```

Comment:
Is this change intended?

Reply:
I was trying to fix the build before archiving. Turned out not to be easy. I'll revert this.